### PR TITLE
make global css work

### PR DESCRIPTION
**Pull request by LucaMele**

### CODE REVIEW

File: postcss.config.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  plugins: [
+    require('autoprefixer')({ // eslint-disable-line
+
+    }),
+    require('./stack/tasks/postcss-font-base64.js')({ // eslint-disable-line
```
Comment (AndyOGo) [inline postcss.config.js]: I would provide it as a separate module in the future. For now is okay.

File: src/index.html [hunk 2]
```diff
@@ -23,6 +23,7 @@
     <script src="app/webcomponents-loader.js"></script>
     <script src="app/app.js"></script>
     <link rel="stylesheet" href="./index.css">
+    <link rel="stylesheet" href="./sg.css">
```
Comment (AndyOGo) [inline src/index.html]: awesome

File: src/sg.scss [hunk 3]
```diff
@@ -0,0 +1 @@
+@import "app/preview";
```
Comment (AndyOGo) [inline src/sg.scss]: awesome again

File: postcss.config.js (hunk 1 above)
Comment (LucaMele) [reply postcss.config.js]: i though that as well. Totally agree

File: src/index.html (hunk 2 above)
Comment (LucaMele) [reply src/index.html]: :)

**Comment by LucaMele** [comment]
fixes #158

**Pull request merged**